File: BankingSystemFrontEnd/bankingsystem/src/ADMIN/Account.js
import React from 'react'
import './Admin.css'

export default function Account(props) {
    return (

        <div id='account'>
            <ul>
                <li>
                    <div id="accountbox">
                        <h1> {props.accountNumber}</h1>
                        <h1> {props.balance}</h1>
                        <h1> {props.createdAt}</h1>
                        <h1>{props.accountType}</h1>
                    </div>
                </li>
            </ul>
        </div>

    )
}

File: BankingSystem/src/main/java/edu/miu/bankingsystem/domain/Account.java
package edu.miu.bankingsystem.domain;

import com.fasterxml.jackson.annotation.JsonBackReference;
import com.fasterxml.jackson.annotation.JsonManagedReference;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.util.List;

@Entity
@AllArgsConstructor
@NoArgsConstructor
@Data
public class Account {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    private String accountNumber;
    private double balance;
    private LocalDate createdAt;
    private String accountType;

    @ManyToOne
    @JoinColumn(name = "users_id")
    @JsonBackReference
    private Users users;

    @OneToMany(mappedBy = "account" ,cascade = CascadeType.ALL, fetch = FetchType.LAZY)
    @JsonManagedReference
    private List<Transaction> transactions;

}
